lib/form/js/validation.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-env browser */
+/* global allPlugins, allInputs, allSettings, allIDs */
+
+// TODO: Add In-Browser Tests
+(function formValidation() {
+  var valueCheck = function (input, settings, validation) {
+    if (input.target.required && (input.target.required === 'save' || input.target.required === 'publish') && input.target.value === '') {
+      return 'Field cannot be left blank!';
+    }
+
+    return validation(input, settings);
+  };
+
+  // Validation Function
+  var validate = function (e) {
```

Comment:
That should be handled in your PR. This is just to split them out in to different files